index.js

```diff
@@ -0,0 +1,22 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let button = document.createElement("button");
+    button.id="clearBtn";
+    button.innerText = "Clear CSS";
+    document.body.appendChild(button);
+    button.addEventListener("click", () => {
+        let link = document.querySelector("#cssLink");
+        if(link.href === "file:///Users/kevinwong/Desktop/Pursuit/Assignments/Unit%202%20-%20Labs/Pursuit-Core-CSS-Intro-Lab/master.css"){
```

Comment:
You can also check if the href is truthy, then if it is then remove the href to something else. otherwise add it back.